knowledge_panels/ingredients/world-en.yml

```diff
@@ -0,0 +1,16 @@
+groups:
+  - 
+    tag_type: ingredients
+    country: fr
+    lang: fr
```

Comment:
not sure what's the effect of country / language, if it's a filter, then I guess we want only lang:  , and it should be "en" I guess

Reply:
Yes my bad, the target is clearly French and France, I'll translate ASAP

Reply:
It allows to have country specific description, by default the world is used